spyder/widgets/mixins.py

```diff
@@ -318,7 +327,8 @@ def handle_sub(matchobj):
             formatted_lines = []
             name = signature.split('(')[0]
             indent = ' ' * (len(name) + 1)
-            rows = textwrap.wrap(signature, width=60, subsequent_indent=indent)
+            rows = textwrap.wrap(signature, width=width,
+                                 subsequent_indent=indent)

```

Comment:
We might want to have a different width for the signatures, it might look better to have the wrapping for the signature smaller than the wrapping for the docstring.

Thoughts @ccordoba12 and @dalthviz ?

Reply:
Having the width of the signatures smaller than the one for the documentation looks something like this:

![diff](https://user-images.githubusercontent.com/16781833/59576500-ad0ef700-9085-11e9-8e51-5856062d2899.png)

Having it the same looks something like this:

![same](https://user-images.githubusercontent.com/16781833/59576542-d891e180-9085-11e9-9ba7-1adb121497c0.png)

Reply:
I kind like it better with the signature and documentation having the same width but it's not much of a difference for me